test/e2e/treatments.e2e.test.ts

```diff
@@ -0,0 +1,92 @@
+import 'dotenv/config.js'
+import * as mwoffliner from '../../src/mwoffliner.lib.js'
+import domino from 'domino'
+import rimraf from 'rimraf'
+import { execa } from 'execa'
+import { jest } from '@jest/globals'
+import { renderers } from './rendererList.js'
+import { zimcheckAvailable, zimdumpAvailable, zimcheck, zimdump } from '../util.js'
+
+jest.setTimeout(200000)
+
+let zimcheckIsAvailable
+let zimdumpIsAvailable
+
+beforeAll(async () => {
+  zimcheckIsAvailable = await zimcheckAvailable()
+  zimdumpIsAvailable = await zimdumpAvailable()
+})
+
```

Comment:
If we could avoid to do that for each test, this would be really better. No way to do simpler?

Reply:
This `beforeAll` hook is placed outside `describe` blocks, so it will execute only once.

Reply:
The problem is not the execution time, but that we have to place this piece of code almost in all e2e test.

Reply:
It's not an execution problem, it's a code problem, these lines should be only once coded in the whole codebase.

Reply:
Done in **test/testAllRenders.ts**